### PR TITLE
Javascript/javascript3/week2

### PR DESCRIPTION
**Pull request by dinythomas89**
done with Js3/week2

### CODE REVIEW

File: javascript/javascript3/week2/index.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        }, resolveAfter * 1000);
+    })
+    return promise;
+}
+promiseFunctionForSetTime(3).then(() => {
+    console.log('I am called asynchronously');
+})
+
+//using async/await
+async function asyncFunctionForSetTime(resolveAfter) {
```
Comment (senner007) [inline javascript/javascript3/week2/index.js:28]: This won't work. You will have to return a promise from the async function

File: javascript/javascript3/week2/index.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+promise
+    .then((response) => response.json())
+    .then((yesOrNo) => {
+        console.log(yesOrNo.answer);
+    })
+
+//using async/await
+async function fetchingData() {
+    try {
+        await promise;
```
Comment (senner007) [inline javascript/javascript3/week2/index.js]: line 91 should be removed right?

**Review by senner007 — APPROVED** [review]
Good :)
Check the comments

File: javascript/javascript3/week2/index.js (hunk 2 above)
Comment (dinythomas89) [reply javascript/javascript3/week2/index.js]: Thanks Niels..done as you suggested..

**Pull request merged**